src/components/Integrations/Shopify/Shopify.css

```diff
@@ -1,8 +1,5 @@
-img.shopify-logo {
-    width: 80px;
-}
 @media only screen and (max-width: 600px) {
-    img.shopify-logo {
-        display: none;
-    }
-}
\ No newline at end of file
+  img.shopify-logo {
+    display: none;
+  }
+}
```

Comment:
You will completely remove this file, right?

Reply:
No, because I need to add the media query first in the styled component. Then I will remove the file.

Reply:
i don't understand why use mediaquery 600px. we support mobile?

Reply:
If I reduce the size of the window, the icon is like this:
![image](https://user-images.githubusercontent.com/2439363/61394870-7f8eb680-a89a-11e9-9fc3-0a79bd84bbd4.png)
That's why I think would be better to hide it.